Common/TableProducer/PID/pidTPC.cxx

```diff
@@ -135,7 +136,8 @@ struct tpcPid {
     }
   }
 
-  void process(Coll const& collisions, Trks const& tracks)
+  void process(Coll const& collisions, Trks const& tracks,
+               aod::BCsWithTimestamps const&)
```

Comment:
Where is the Timestamp used later in the task? Or do we need to implement this?

Reply:
I will add it in another PR, I wanted to set this first, this is what I mentioned to you yesterday that is missing

Reply:
Ah I see. This answers my question thanks!